docs/src/assets/scss/components/rules.scss

```
@@ -45,8 +50,9 @@
 .rule-category {
     font-size: var(--step--1);
     display: flex;
+    flex-direction: column;
+    height: 8.5rem;
```

```suggestion
    flex-wrap: wrap;
```
Set `height: 100%` on the `.rule-category .rule-category__description` class to properly align the text.

---

Done, thanks!